code/PostgreSQLSchemaManager.php

```diff
@@ -560,8 +566,17 @@ private function alterTableAlterColumn($tableName, $colName, $colSpec)
 
     public function renameTable($oldTableName, $newTableName)
     {
+        $constraints = $this->getConstraintForTable($oldTableName);
         $this->query("ALTER TABLE \"$oldTableName\" RENAME TO \"$newTableName\"");
+
+        if ($constraints) {
+            foreach ($constraints as $old) {
```

Comment:
The rest of our logic assumes that constraint names start with the same name as the table. This was causing us issue with the `_Versions` tables. The SS3 equivalent was using `_versions` all lowercase.

When we looking to see if the constraints needed updating, we were looking for the uppercase version, not realising that the lowercase version were still there.